### PR TITLE
Add resource cleanup instructions for several event-source examples

### PR DESCRIPTION
**Pull request by lberk**
### Proposed Changes

- cronjob-source: cleanup event source, service
- iot-core: cleanup gcpubsub, channel, gcpubsub-source and subscription
- kubernetes-event-source: cleanup channel, service account, k8s event, and subscription

### CODE REVIEW

File: docs/eventing/samples/iot-core/README.md [hunk 1]
```diff
@@ -201,3 +201,35 @@ see them in the subscriber.
...
         -events 10
     ```
+
+### Cleanup
+
+To cleanup the knative resources:
+
+1.  Remove the gcppubsub controller:
+    ```shell
+    pushd $HOME/go/src/github.com/knative/eventing-sources
```
Comment (gyliu513) [inline docs/eventing/samples/iot-core/README.md]: Instead of using `pushd`, how about use `cd` instead?

Comment (lberk) [reply docs/eventing/samples/iot-core/README.md]: We can do that, but I wanted to be consistent with the setup instructions.  I think it'd make sense to use the same (whichever that is) in both places.  What would you prefer?

Comment (gyliu513) [reply docs/eventing/samples/iot-core/README.md]: +1 to keep consistent, but I'd prefer that we use `cd` instead. I think we can handle this in other PR.

**Comment by lberk** [comment]
@RichieEscarez Hi! I think this is a relatively simple PR, is there anything else you'd like to see before getting an /approved ?

**Comment by matzew** [comment]
@samodell I think it's /approve :)

**Pull request merged**